crypto/vrf/README.md

```diff
@@ -25,17 +25,20 @@ Use `func init()` with `build` option
       * `//go:build libsodium`
       * `// +build !libsodium,!coniks`
       * `func init() { defaultVrf = newVrfEd25519r2ishiguro() }`
+      * `const ProofSize = 81`
       * vrf_r2ishiguro.go
     * (coniks)
       * `//go:build coniks`
       * `// +build coniks`
       * `func init() { defaultVrf = newVrfEd25519coniks() }`
+      * `const ProofSize = 96`
       * vrf_coniks.go
       * vrf_coniks_test.go
     * (libsodium)
       * `//go:build libsodium`
       * `// +build libsodium`
       * `func init() { defaultVrf = newVrfEd25519libsodium() }`
+      * `const ProofSize = int(libsodium.PROOFBYTES)`
```

Comment:
I want to know bytes value. How about adding following comments?
```markdown
* `const ProofSize = int(libsodium.PROOFBYTES) // 80 bytes`
```

Reply:
I think adding this comment dilutes the benefits of importing fixed values ​​from libsodium.
The comment should be amended when this value changes. What do you think?

Reply:
I just thought it would be useful to know the value of bytes like any other vrf library files.

Reply:
OK, It's fine as this is.

Reply:
Thanks anyway.